packages/cashier/src/modules/deposit-fiat/deposit-fiat.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { PageContainer } from '../../components/page-container';
+import { DepositFiatIframe } from './components';
+
+const DepositFiat: React.FC = () => (
+    <PageContainer>
+        <DepositFiatIframe />
+    </PageContainer>
+);
+export default DepositFiat;
```

Comment:
```suggestion
);

export default DepositFiat;
```
@aum-deriv Always leave a blank line before the export statement.